### DISCUSSION TITLE
Why lcurve.out have no information for testing like `l2_e_tst`, `l2_e_tst` in V2.0.0 ?

### DISCUSSION THREAD

**Discussion by zhao-w-en** [deepmodeling/deepmd-kit#1243] [Q&A]
In V2.0.0 lcurve.out the headline is only `#  step      rmse_trn    rmse_e_trn    rmse_f_trn         lr` , while V1.3.3 has `# batch      l2_tst    l2_trn    l2_e_tst  l2_e_trn    l2_e_tst  l2_f_trn         lr`

**Comment by njzjz** [comment]
In v2.0 you need to set `validation_data` manually.

**Reply by zhao-w-en** [reply]
Thanks for your reply! 
 I remember that the last set of train system is used for testing in V1.3.3, so does that mean validation_data is corresponding to that set ? `numb_test ` is number of frames used for the test during training. What is the relationship between `batch_size` of `validation_data` part and `numb_test ` ?

**Reply by njzjz** [reply]
> does that mean validation_data is corresponding to that set ? 

Yes.

**Reply by njzjz** [reply]
> What is the relationship between batch_size of validation_data part and numb_test

Good question. I think `numb_test` has been deprecated? cc @wanghan-iapcm

**Reply by wanghan-iapcm** [reply]
In v2.x the `numb_test` does not have any effect.

**Reply by zhao-w-en** [reply]
Oh, thank you! By the way, I also have some questions on `numb_btch` and `batch_size` of validation part. If `numb_bath` determine how many systems will be used for validation, `batch_size` determine what? 
PS. maybe you could remark the parameter that it only suit for V1.3.3 from `https://docs.deepmodeling.org/projects/deepmd/en/latest/train/train-input.html`.
